web/themes/joinup/templates/navigation/menu-local-tasks.html.twig

```diff
@@ -12,10 +12,16 @@
  */
 #}
 {% if primary %}
+  <button id="local-tasks-primary" class="mdl-button mdl-js-button mdl-button--icon">
```

Comment:
I didn't find an example when secondary links are displayed, but if it is in default drupal template I don't won't to remove it. I think that in your code if there are primary and secondary links, then only primary would be displayed (because there is if .. if, not if .. elseif )